Symfony/CS/Fixer/PSR1/Psr4Fixer.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\PSR1;
```

Comment:
OK, I see you change the description that I wasn't sure about.
Then indeed this is part of psr0 as well as psr4. could be psr0, but then it would be great to name a fixer after what it fixes, `psr4` is so meaningless as a fixer name.